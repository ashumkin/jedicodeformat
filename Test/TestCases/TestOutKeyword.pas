unit TestOutKeyword;

{ AFS
 This unit compiles but is not semantically meaningfull
 it is test cases for the code formatting utility
 Test the "out" keyword, or abuse of it
}

interface

type
  TMyClass = class
    Out : integer;
  end;


implementation

end.
