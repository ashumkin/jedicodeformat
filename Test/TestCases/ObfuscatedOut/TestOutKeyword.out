unit testoutkeyword;

interface

type
  tmyclass = class
    out: integer;
  end;

implementation

end.
 